pvlib/spa.py

```diff
@@ -835,24 +835,24 @@ def equation_of_time(sun_mean_longitude, geocentric_sun_right_ascension,
     return E
 
 
-@jcompile('void(float64[:], float64[:], float64[:,:])', nopython=True,
-          nogil=True)
-def solar_position_loop(unixtime, loc_args, out):
+@jcompile('void(float64[:], float64[:], float64[:], float64[:,:])',
+          nopython=True, nogil=True)
+def solar_position_loop(unixtime, delta_t, loc_args, out):
```

Comment:
Whereas `delta_t` used to be a scalar that could live in `loc_args`, now it is a time-varying value (same length as `unixtime`) and thus requires a separate array input.  So a new parameter is needed, and `loc_args` shrinks by one.